mesonbuild/interpreter/interpreter.py

```diff
@@ -1135,6 +1135,17 @@ def func_project(self, node: mparser.FunctionNode, args: T.Tuple[str, T.List[str
         if self.build.project_version is None:
             self.build.project_version = self.project_version
         proj_license = kwargs['license']
+        if isinstance(proj_license, list):
+            triggered = FeatureDeprecated.single_use(
+                'project license as an array of strings', '0.62.0', self.subproject,
+                'Use an SPDX license expression with AND and/or OR instead', node)
+            if not triggered and len(proj_license) > 1:
+                mlog.warning(
+                    'Passing an array of licenses to project() is ambiguous, '
+                    'consider using an SPDX license expression string instead.',
```

Comment:
I don't like the warning message being repeated, why not passing an new arg to `FeatureDeprecated.single_use()` like `always_warn=True`? Note that Meson already print a NOTICE in the summary about future deprecations, that could be enough?

Reply:
I already had this same discussion in the previous review on this PR, although my opinion there was that we should only use mlog.warning and not use FeatureDeprecated at all.

Reply:
It's in the review comment above indeed. Closing this discussion.